samples/sampler/.qa-storybook/preview.js

```diff
@@ -48,6 +48,7 @@ export const parameters = {
 		container: DocsContainer,
 		page: DocsPage,
 		iframeHeight: 360,
+		inlineStories: false,
```

Comment:
Should we do the same as https://github.com/enactjs/sandstone/blob/develop/samples/sampler/.storybook/preview.js#L57 for moonstone too?

Reply:
we have WRP-4945 which is just waiting for this PR to be merged :)